functions-and-operators/string-functions.md

```diff
@@ -179,11 +179,198 @@ Return the length of a string in bytes.
 
 ### [`LIKE`](https://dev.mysql.com/doc/refman/8.0/en/string-comparison-functions.html#operator_like)
 
-Simple pattern matching.
+The `LIKE` operator is used for simple string matching, the format `expr LIKE pat [ESCAPE 'escape_char']` returns `1(TRUE)` or `0(FALSE)`
```

Comment:
Might be good to mention how collations affect `LIKE` and that there is `ILIKE` and `REGEXP` for similar tasks in TiDB as well.
```
mysql> SELECT '🍣🍺sushi🍣🍺' COLLATE utf8mb4_bin LIKE '%sushi%';
+------------------------------------------------------------+
| '🍣🍺sushi🍣🍺' COLLATE utf8mb4_bin LIKE '%sushi%'                 |
+------------------------------------------------------------+
|                                                          1 |
+------------------------------------------------------------+
1 row in set (0.00 sec)

mysql> SELECT '🍣🍺Sushi🍣🍺' COLLATE utf8mb4_bin LIKE '%sushi%';
+------------------------------------------------------------+
| '🍣🍺Sushi🍣🍺' COLLATE utf8mb4_bin LIKE '%sushi%'                 |
+------------------------------------------------------------+
|                                                          0 |
+------------------------------------------------------------+
1 row in set (0.00 sec)

mysql> SELECT '🍣🍺Sushi🍣🍺' COLLATE utf8mb4_unicode_ci LIKE '%sushi%';
+-------------------------------------------------------------------+
| '🍣🍺Sushi🍣🍺' COLLATE utf8mb4_unicode_ci LIKE '%sushi%'                 |
+-------------------------------------------------------------------+
|                                                                 1 |
+-------------------------------------------------------------------+
1 row in set (0.00 sec)

mysql> SELECT '🍣🍺Sushi🍣🍺' COLLATE utf8mb4_bin ILIKE '%sushi%';
+-------------------------------------------------------------+
| '🍣🍺Sushi🍣🍺' COLLATE utf8mb4_bin ILIKE '%sushi%'                 |
+-------------------------------------------------------------+
|                                                           1 |
+-------------------------------------------------------------+
1 row in set (0.00 sec)

mysql> SELECT '🍣🍺Sushi🍣🍺' REGEXP '.*sushi.*';
+--------------------------------------------+
| '🍣🍺Sushi🍣🍺' REGEXP '.*sushi.*'                 |
+--------------------------------------------+
|                                          1 |
+--------------------------------------------+
1 row in set (0.01 sec)
```

Reply:
Add the example using `COLLATE`
I thought it's better only `LIKE` because this section mentions about itself
https://github.com/pingcap/docs/pull/16084/commits/4a9572539ed773ea0d8e2b6faa40463b0a49bdce#diff-ca49915f7ccf3346527f215e50d580ad5df1526e72f8c7cdcde992500384239aR277-R303